packages/story-editor/src/components/library/panes/pageTemplates/defaultTemplateList.js

```diff
@@ -37,19 +37,28 @@ import { PANE_PADDING } from '../shared';
 import { useConfig } from '../../../../app/config';
 
 import DefaultPageTemplate from './defaultPageTemplate';
+import SavedPageTemplate from './savedPageTemplate';
 
 const WrapperGrid = styled.div`
   display: grid;
   width: 100%;
   margin-left: ${PANE_PADDING};
+  margin-bottom: 12px;
```

Comment:
Noticed there was no spacing at the end of the template grids, so added some margin so that it doesn't look cutoff.